packages/terra-date-time-picker/src/DateTimeUtils.js

```diff
@@ -141,7 +141,7 @@ class DateTimeUtils {
       return false;
     }
 
-    const localizedDateTime = moment.tz(dateTime.format(), moment.tz.guess());
+    const localizedDateTime = moment.tz(dateTime.format(), dateTime.tz());
```

Comment:
is `dateTime` guaranteed that it is in the correct time zone?

Reply:
Yes it is.